### DISCUSSION TITLE
Welcome to DoubleML Discussions!

### DISCUSSION THREAD

**Discussion by MalteKurz** [DoubleML/doubleml-for-py#133] [General]
## 👋 Welcome to DoubleML Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas :bulb:.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jaschwer** [comment]
Thank you for this very helpful R package! I was wondering if the sensitivity analysis that is already available for the Python code can also be done in R? I didn't see it in the example or in the R help file. If not, is there any timetable/idea by when this should be doable? I would like to do this sensitivity analysis without switching to Python.

**Reply by PhilippBach** [reply]
Dear @jaschwer 

unfortunately, sensitivity analysis is currently only available in the Python version. The implementation should actually not be too complicated for R, as we already have the Python code as a template... We plan to integrate it but cannot really say something about the timing. 

This year we wanted to update the R package & integrate sensitivity analysis (ad also heterogeneous treatment effects), but as I said it's hard to say when this will be... In case you want to help with the implementation, we would appreciate any help
